raft/server.cc

```diff
@@ -626,8 +637,8 @@ future<> server_impl::wait_for_entry(entry_id eid, wait_type type, seastar::abor
     }
     SCYLLA_ASSERT(inserted);
     if (as) {
-        it->second.abort = as->subscribe([it = it, &container] () noexcept {
-            it->second.done.set_exception(request_aborted());
+        it->second.abort = as->subscribe([it = it, &container]() noexcept {
```

Comment:
nit: why remove the whitespace? We usually include such whitespaces

Reply:
Clang format did that, anyway removed parenthesis!